cfn-resources/project/test/cfn-test-create-inputs.sh

```diff
@@ -1,26 +1,12 @@
-#!/bin/bash
-
-# Copyright 2023 MongoDB Inc
-#
-# Licensed under the Apache License, Version 2.0 (the "License");
-# you may not use this file except in compliance with the License.
-# You may obtain a copy of the License at
+#!/usr/bin/env bash
+# cfn-test-create-inputs.sh
 #
-#      http://www.apache.org/licenses/LICENSE-2.0
-#
-# Unless required by applicable law or agreed to in writing, software
-# distributed under the License is distributed on an "AS IS" BASIS,
-# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
-# See the License for the specific language governing permissions and
-# limitations under the License.
-
-# ./cfn-test-create-inputs.sh PROJECT_NAME
 # This tool generates json files in the inputs/ for `cfn test`.
+#
 
-set -euo pipefail
 
 function usage {
-	echo "usage:$0 <project_name>"
+    echo "usage:$0 <project_name>"
```

Comment:
you don't have the commit-hook set in your machine. You should run `make link-git-hooks` in the root of the repository so that the committed files will be automatically formatted.

Reply:
Thanks @andreaangiolillo! that helps!